### DISCUSSION TITLE
[18 gennaio 2023] Esercizio 6

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/psd2223#232] [Esami]
Data la funzione

## fila a

$$
    \Large f = \overline{a}d \oplus (a\overline{b} + bc)
$$

## fila b

$$
    \Large f = \overline{b}c \oplus (\overline{a}b + ad)
$$

rappresentarla in forma POS specificando assiomi e regole dell’algebra di Boole utilizzati.

> [!NOTE]
> specificare gli assiomi e le regole dell’algebra di Boole utilizzati è richiesto solo nella traccia del canale M-Z
(nella traccia del canale A-L semplicemente di "utilizzarli")

<details closed>
    <summary> soluzione A-L fila A </summary>
    
<div align="center">

![image](https://github.com/user-attachments/assets/dd6a59e6-6834-46da-9b98-6ba71777ff5b)

</div>
</details>

**Comment by Dieal** [comment]
![2025-01-08-11-54-20-800_1](https://github.com/user-attachments/assets/7ec50fc0-79c3-4acd-8a09-42ff336dc51f)
Per chi non capisse i passaggi della soluzione
